pkgs/data/icons/buuf-icon-theme/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenvNoCC, fetchzip }:
```

Comment:
I'm not able to build this package because it's not listed in `pkgs/top-level/all-packages.nix`. There is however a new paradigm on it's way in right now, where if you put the package in `pkgs/by-name/bu/buuf-icon-theme/package.nix`, it will be included in `pkgs` automatically. While it's not required just yet, feel free to try it out!